tests/model_executor/model_loader/instanttensor_loader/test_weight_utils.py

```
@@ -160,6 +213,385 @@ def test_instanttensor_restricts_io_to_indexed_shards(tmp_path):
+    regular_tensor = torch.tensor([1.0, 2.0])
+    priority_tensor = torch.tensor([3.0, 4.0])
+    save_file({"model.weight": regular_tensor}, regular_shard)
+    save_file({"vision_tower.weight": priority_tensor}, priority_shard)
+
+    regular_metadata = _safetensors_tensor_metadata(regular_shard)["model.weight"]
+    priority_metadata = _safetensors_tensor_metadata(priority_shard)[
+        "vision_tower.weight"
+    ]
+
+    class FakeReader:
+        filename = [str(regular_shard), str(priority_shard)]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)